extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,241 @@
+          "type": "string",
+          "format": "date-time",
+          "description": "Conversion Time of the Event"
+        },
+        "xdm:triggerTime": {
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Trigger Time of the Event"
+        },
+        "xdm:originalTriggerTime": {
+          "title": "Original Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Original Trigger Time of the Event"
+        },
+        "xdm:advertiserId": {
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Identifier for the advertiser"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name"
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign Identifer on the advertising platform"
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform. It will be populated only for non-search advertising related campaigns"
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform"
+        },
+        "xdm:adId": {
+          "title": "Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the advertising platform"
+        },
+        "xdm:term": {
+          "title": "Search Term",
+          "type": "string",
+          "description": "Search Term that contributed to the conversion"
+        },
+        "xdm:termId": {
+          "title": "Search Term Identifier",
+          "type": "string",
+          "description": "Search Term ID on the search advertising platform"
+        },
+        "xdm:matchType": {
+          "title": "Term Match Type",
+          "type": "string",
+          "description": "Match Type associated to the Search Term being targeted for the Ad",
+          "enum": [
+            "Broad",
+            "Phrase",
+            "Exact",
+            "Advanced",
+            "Standard",
+            "Website",
+            "Product Target",
+            "Product Group",
+            "unknown"
+          ],
+          "meta:enum": {
+            "Broad": "Broad",
+            "Phrase": "Phrase",
+            "Exact": "Exact",
+            "Advanced": "Advanced",
+            "Standard": "Standard",
+            "Website": "Website",
+            "Product Target": "Product Target",
+            "Product Group": "Product Group",
+            "null": "Unknown"
+          }
+        },
+        "xdm:domainPlacement": {
```

Comment:
If this means the URL of the Publisher, can you please add that to the `title` and `description` ?